MonoGame.Framework/Content/ContentReaders/DictionaryReader.cs

```diff
@@ -7,9 +7,14 @@
 using MonoGame.Framework.Utilities;
 
 namespace Microsoft.Xna.Framework.Content
-{
-
-    internal class DictionaryReader<TKey, TValue> : ContentTypeReader<Dictionary<TKey, TValue>>
+{
+    /// <summary>
+    /// This type is not meant to be used directly by MonoGame users.
+    /// Its purpose is to allow to work-around AOT issues when loading assets with the ContentManager fail due to the absence of runtime-reflection support in that context (i.e. missing types due to trimming and inability to statically discover them at compile-time).
+    /// If ContentManager.Load() throws an NotSupportedExeception, the message should provide insights on how to fix it.
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.All)]

```

Comment:
@ThomasFOG  I this has been merged, but any particular reason why we are using the Full class namespace there rather than adding a `using` clause?

Reply:
No particular reason. Just my personal coding style which is "if it's used only once, avoid a using".